crates/router/src/types/domain/user.rs

```diff
@@ -988,18 +1001,6 @@ impl UserFromStorage {
                 created_at: common_utils::date_time::now(),
             };
 
-            #[cfg(feature = "keymanager_create")]
-            {
-                common_utils::keymanager::create_key_in_key_manager(
-                    key_manager_state,
-                    EncryptionCreateRequest {
-                        identifier: Identifier::User(key_store.user_id.clone()),
-                    },
-                )
-                .await
-                .change_context(UserErrors::InternalServerError)?;
-            }
-
             state
                 .global_store
                 .insert_user_key_store(key_manager_state, key_store, &master_key.to_vec().into())
```

Comment:
What will happen to transferred key if the db insertion fails here? Will it also be deleted?

Reply:
No that won't be removed.

Reply:
Will that doesn't cause inconsistency?

Reply:
No, if new user store created for the same user it will be updated in the key store

Reply:
Discussed with @ArjunKarthik, came to a conclusion that this function is temporary and transfer will override the service.